interactive/plugin.py

```diff
@@ -95,7 +95,7 @@ def gen_nodes(item, cache):
     chain = item.listchain()
     for node in chain:
         try:
-            name = node._obj.__name__
+            name = node.obj.__name__
```

Comment:
@vodik nice. Yeah only thing is do we need to pin **after** a particular `pytest` version now?